load_test.py

```diff
@@ -0,0 +1,96 @@
+import time
+from sdcm.tester import ClusterTester
+from sdcm.db_stats import PrometheusDBStats
+
+
+class LoadTest(ClusterTester):
+    """
+    Test Scylla load with cassandra-stress
+    """
+
+    def run_load_and_check_for_admission_control(self, prepare_base_cmd, stress_base_cmd, job_num=8):  # pylint: disable=invalid-name
```

Comment:
why is 8 in the default for the function?
remove it.
why is pylint complain about invalid-name?

Reply:
because it is the maximum the i3.xlarge machine can run (8 cores).
because the snake case name is "too long"

Reply:
changed